mldsa/fips202/fips202.c

```diff
@@ -586,23 +625,16 @@ __contract__(
   assigns(memory_slice(s, sizeof(uint64_t) * MLD_KECCAK_LANES))
   assigns(memory_slice(out, nblocks * r)))
 {
-  unsigned int i;
-
   while (nblocks > 0)
   __loop__(
-    assigns(i, out, nblocks, memory_slice(s, sizeof(uint64_t) * MLD_KECCAK_LANES), memory_slice(out, nblocks * r))
+    assigns(out, nblocks, memory_slice(s, sizeof(uint64_t) * MLD_KECCAK_LANES),
+      memory_slice(out, nblocks * r))
     invariant(nblocks <= loop_entry(nblocks))
-    invariant(out == loop_entry(out) + r * (loop_entry(nblocks) - nblocks) && nblocks >= 0)
```

Comment:
Out of curiosity, why was this part of the invariant removed? `&& nblocks >= 0`

Reply:
It could be skipped. It is not used in the mlkem-native proof
https://github.com/pq-code-package/mlkem-native/blob/ae2afe58380797f6b6789060b3ff8be2b2917fb5/mlkem/fips202/fips202.c#L131
so I assume we prefer to omit it here as well.